src/test/java/uk/gov/hmcts/reform/idam/service/IdamUserDisposerServiceTest.java

```diff
@@ -60,8 +60,8 @@ void shouldRunAtLeastOnce() {
         verify(staleUsersService, times(1)).fetchStaleUsers();
         verify(userRoleService, times(1)).filterUsersWithRoles(any());
         verify(deleteUserService, times(1)).deleteUsers(any());
-        verify(loggingSummaryUtils, times(1)).logSummary(anyLong(),anyLong(),
-                              eq(3),eq(2));
+        verify(loggingSummaryUtils, times(1))
```

Comment:
We need a test to confirm that the disposer will continue running even if idam deletion fails. This test class might not be the right place though.